Sources/Metaplex/Programs/Tokens/MetaplexTokenProgram.swift

```diff
@@ -8,7 +8,7 @@
 import Foundation
 import Solana
 
-public class TokenProgram {
+public class MetaplexTokenProgram {
```

Comment:
This had to be renamed due to namespacing issues.

Since we already have `TokenProgram` in Solana and we are now trying to use `TokenProgram` to create instructions we had a conflict. Namespacing with `Solana.TokenProgram` wouldn't work either since `Solana` is an object within `Solana`.

We need to do proper deprecation/warnings here, but want to see your thoughts before doing so.